api-reference/10 UI Components/dxGantt/3 Methods/exportToPdf(options).md

```diff
@@ -3,13 +3,81 @@ id: dxGantt.exportToPdf(options)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports Gantt chart data to a PDF document.
 
 ##### return: Promise<any>
-<!-- Description goes here -->
+A Promise that is resolved after the Gantt chart is exported. It is a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">native Promise</a> or a <a href="http://api.jquery.com/Types/#Promise" target="_blank">jQuery.Promise</a> when you use jQuery.
 
 ##### param(options): Object
-<!-- Description goes here -->
+PDF export options.
+
+---
+
+The **export** method allows you to save information about the Gantt chart's layout, appearance, and tasks. This method supports the following options:
+
+- **format** {string | width?: number, height?: number } - Specifies the document size.
+- **landscape** {boolean} - Specifies whether to use horizontal orientation for the document.
+- **fileName** {string} - Specifies the file name.
+- **exportMode** {"all", "treeList", "chart"} -  Specifies which part of the component to export (chart area, tree list area or the entire component).
+- **dateRange**: {"all" | "visible" | startDate? : Date, endDate? : Date, startIndex? : number, endIndex? : number } - Specifies the date range for which to export tasks.
+- **margins** { left?: number, top?: number, right?: number, bottom?: number } - Specifies the outer indents of the exported area.
+
+---
+
+How to export the Gantt chart to a PDF document with settings:
```

Comment:
This sounds more like the title of an article. Can we make it more specific, so that it relates to the code (markup?) below? For example, do you mean:

`Use the following settings to export the Gantt chart to a PDF document:`

Reply:
I want to name these lines as headings and then use them in the Example article (in future). If this sounds bad, then I will use your version. What do you think?

Reply:
Now, I've applied your suggestion.

Reply:
Thank you, Anna!